pages/multisearch/[tcgPath].tsx

```
@@ -0,0 +1,348 @@
+              tcg={tcg}
+              setTcg={setTcg}
+              searchInput={searchInput}
+              setSearchInput={setSearchInput}
+              handleSubmit={handleSubmit}
+              vendors={vendors}
+              selectedWebsites={selectedWebsites}
+              onWebsiteSelect={onWebsiteSelect}
+              minimumAcceptableCondition={minimumAcceptableCondition}
+              setMinimumAcceptableCondition={setMinimumAcceptableCondition}
+            />
+          </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)